src/components/Results.tsx

```diff
@@ -65,19 +64,17 @@ export const Results: React.FC = () => {
   }, [mode]);
 
   // handle mobile list scrolling
-  const handleScroll = (e: React.UIEvent<HTMLDivElement, UIEvent>) => {
-    if (e.currentTarget.scrollTop > 10) {
-      setMode(ResultsMode.list);
-    }
-    if (e.currentTarget.scrollTop === 0) {
-      if (preventMapView) {
-        setPreventMapView(false);
-        return;
-      }
-      setMode(ResultsMode.map);
-      setPreventMapView(true);
-    }
-  };
+  // const handleScroll = useCallback(
```

Comment:
can't this be removed? If so, can you include a comment why it is commented and when it should be included again?